static/StashpointTemplate.json

```diff
@@ -435,12 +435,6 @@
                                                                                 },
                                                                                 "do": {
                                                                                     "$if ": {
-                                                                                        "$condition": {
-                                                                                            "$or": [
-                                                                                                "$not $eqs ($.Item.Unlockable.Id,TOKEN_OUTFIT_WET_SUIT)",
-                                                                                                "$not $eqs ($arg location,LOCATION_NEWZEALAND)"
-                                                                                            ]
-                                                                                        },
```

Comment:
I wonder why this code is in the template. In theory, there should be 2 wetsuits: one which is unlockable, and one which is the one that the map uses by default.

Reply:
Also this is not how you remove an if statement, I'm surprised this doesn't crash.